addons/portal/static/src/scss/portal.scss

```diff
@@ -300,12 +306,11 @@ img.o_we_custom_image {
 // Portal specific
 .o_portal {
     .breadcrumb {
-        background-color: rgba(0, 0, 0, 0);
-        margin-bottom: 0px;
-    }
+        background-color: transparent;
 
-    .record_pager {
-        padding: 8px 0px;
+        .breadcrumb-item {
+            display: inline-flex;
+        }
```

Comment:
Unfortunately yes, at least I didn't find any other solution.  Seems like bts4 breadcrumbs need items to have the same font size/design. Since we customize the active items to act as a title (bigger, bold font weight) the alignment was broken